server.py

```diff
@@ -1,23 +1,76 @@
 import socket
+import email.utils
 
-ADDR = ('127.0.0.1', 7000)
 
-socket = socket.socket(
-    socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
-    )
+def response_ok(uri):
+    returnstr = []
+    content_type = "text/plain"
+    bytes = uri.encode('utf-8')
+    content_length = str(len(bytes))
+    date = email.utils.formatdate(usegmt=True)
+    returnstr.append("HTTP/1.1 200 OK")
+    returnstr.append("Date:" + date)
+    returnstr.append("Content-Type:" + content_type)
+    returnstr.append("Content-Length:" + content_length)
+    header = '\r\n'.join(returnstr)
+    header = header + "\r\n" + uri
+    return header
 
-socket.bind(ADDR)
-socket.listen(1)
 
-while True:
+def response_error():
+    return "500 Server Error \r\n\r\n"
```

Comment:
You are missing quite a bit of the required structure to make this a well-formed HTTP response.